qiskit_nature/problems/second_quantization/vibrational/builders/vibrational_op_builder.py

```diff
@@ -26,45 +24,6 @@
 logger = logging.getLogger(__name__)
 
 
-def _build_vibrational_op(
-    watson_hamiltonian: WatsonHamiltonian,
-    num_modals: Union[int, List[int]],
-    truncation_order: int,
-    basis: Optional[BosonicBasis] = None,
-) -> VibrationalOp:
-    """
-    Builds a :class:`VibrationalOp` based on a :class:`WatsonHamiltonian` object.
-
-    Args:
-        watson_hamiltonian: :class:`WatsonHamiltonian` instance.
-        num_modals: the number of modals per mode.
-        truncation_order: order at which an n-body expansion is truncated
-        basis: the :class:`BosonicBasis` to which the :class:`WatsonHamiltonian` gets converted.
-        Currently, this argument will be ignored until more bases are available. Therefore, it will
-        always use the :class:`HarmonicBasis` internally.
-
-    Returns:
-        VibrationalOp: VibrationalOp built from a WatsonHamiltonian object.
-    """
-    if basis is not None:
-        logger.warning(
-            "The only supported `BosonicBasis` is the `HarmonicBasis`. However you specified "
-            "%s as an input, which will be ignored.",
-            str(basis),
-        )
-
-    num_modes = watson_hamiltonian.num_modes
-
-    if isinstance(num_modals, int):
-        num_modals = [num_modals] * num_modes
-
-    boson_hamilt_harm_basis = HarmonicBasis(
-        watson_hamiltonian, num_modals, truncation_order
-    ).convert()
-
-    return build_vibrational_op_from_ints(boson_hamilt_harm_basis, num_modes, num_modals)
-
-
 @deprecate_function(
```

Comment:
Is this function below (Github won't let me comment further down) still being used somewhere? If yes, why/where? If not we should be able to remove this now, too.

Reply:
Not used anywhere. I removed the file